packages/replay-next/components/console/ErrorStackRenderer.tsx

```diff
@@ -0,0 +1,145 @@
+import { ObjectId, PauseId } from "@replayio/protocol";
+import { ReactNode, Suspense, useContext } from "react";
+
+import { assert } from "protocol/utils";
+import Loader from "replay-next/components/Loader";
+import { getMappedLocationSuspense } from "replay-next/src/suspense/MappedLocationCache";
+import { getObjectPropertySuspense } from "replay-next/src/suspense/ObjectPreviews";
+import { getScopeMapSuspense } from "replay-next/src/suspense/ScopeMapCache";
+import { getSourcesByUrlSuspense } from "replay-next/src/suspense/SourcesCache";
+import { ReplayClientContext } from "shared/client/ReplayClientContext";
+
+import Source from "./Source";
+import styles from "./StackRenderer.module.css";
+
+export default function ErrorStackRenderer({
+  pauseId,
+  errorObjectId,
+}: {
+  pauseId: PauseId;
+  errorObjectId: ObjectId;
+}) {
+  return (
+    <Suspense fallback={<Loader />}>
+      <div className={styles.StackGrid} data-test-name="ErrorStack">
+        <ErrorStackRendererSuspends pauseId={pauseId} errorObjectId={errorObjectId} />
+      </div>
+    </Suspense>
+  );
+}
+
+function ErrorStackRendererSuspends({
+  pauseId,
+  errorObjectId,
+}: {
+  pauseId: PauseId;
+  errorObjectId: ObjectId;
+}) {
+  const client = useContext(ReplayClientContext);
+  const stack = getObjectPropertySuspense(client, pauseId, errorObjectId, "stack").value;
+  assert(typeof stack === "string", "no stack string found in error object");
+  const frames = parseStackString(stack);
```

Comment:
Consider using [the `error-stack-parser` module](https://www.npmjs.com/package/error-stack-parser) instead?